### PR TITLE
cnv: avoid documenting cnv-bridge-cni

### PR DESCRIPTION
**Pull request by dankenigsberg**
As of OpenShift 4.3, the standard bridge-cni and tuning-cni are support
vlan filtering and static mac address, respectfully. We should direct
user to use these standard CNIs and not the CNV-crafted ones.

Signed-off-by: Dan Kenigsberg <danken@redhat.com>

### CODE REVIEW

**Comment by dankenigsberg** [comment]
@duyanyan would you please keep me honest here, verifying that there is no need to use the cnv-crafted CNIs?

**Comment by dankenigsberg** [comment]
@phoracek please approve this PR (or reject it if you think it is wrong)

**Comment by ousleyp** [comment]
LGTM. I will merge if @phoracek and/or @duyanyan approve :)

**Review by ousleyp — APPROVED** [review]

**Comment by phoracek** [comment]
If we want to switch to OpenShift's bridge and tuning plugins, we have to fully honor their API which is not based around NetworkAttachmentDefinition, but around their Network operator. That would require changes in our UI and in docs.

I'm in favor of switching to OCP's CNIs, but this PR would be at the end of a bigger effort. We should not merge this at this point.

**Pull request closed without merging**
